app/livechat/server/lib/Livechat.js

```diff
@@ -781,6 +810,73 @@ export const Livechat = {
 	notifyGuestStatusChanged(token, status) {
 		LivechatInquiry.updateVisitorStatus(token, status);
 		LivechatRooms.updateVisitorStatus(token, status);
+		const room = LivechatRooms.findOne({ t: 'l', 'v.token': token }, { sort: { ts: -1 } });
```

Comment:
Please, add a new helper method for this block of code, it will make the code more readable.